### ISSUE TITLE
Configurable fullscreen resolution does not work

### ISSUE THREAD

**Issue by Fealthas** [ZeroK-RTS/Zero-K#4635]
Hello Dear Devlobs,

Wanted to note that when the settings>graphics> ingame/lobby configurable fullscreen setting is changed, it appears to accept the change but does not actually apply it.

For example, it correctly identifies my 4k screen and sets the resolution to 4k, but when I type in 1920x1080 it accepts this and then continues to render everything in 4k both ingame and out.

**Comment by Sharkgamedev** [comment]
I've been able to reproduce this, however this issue likely belongs over in [Chobby](https://github.com/ZeroK-Team/Chobby), the repo for zk's main menu.

**Comment by GoogleFrog** [comment]
I don't know whether spring is meant to support actual resolution changes for fullscreen. I think this was more a hacky way to do monitor detection. The window mode settings in the engine are currently being looked at, so we'll be able to revisit them in the lobby once the engine seems stable.

**Comment by Fealthas** [comment]
Ok!

Good to see that you are still kicking, sir frog

**Comment by 6AKU66** [comment]
> spring is meant to support actual resolution changes for fullscreen

As i remember it's was supporting this before. But it's was broken somewhere in 400-700 engines.

**Comment by GoogleFrog** [comment]
Yes, I recall ivand did something and claims that window handling works better now. But I haven't been able to get an answer on how the new window handling is meant to be used to replicate all the behaviour we had.

BAR has the same bugs.

**Comment by 6AKU66** [comment]
> Yes, I recall ivand did something and claims that window handling works better now. But I haven't been able to get an answer on how the new window handling is meant to be used to replicate all the behaviour we had.
> 
> BAR has the same bugs. Engine ticket?

Yeah and it's was... closed...
https://github.com/beyond-all-reason/spring/issues/206
